controllers/sessions.js

```diff
@@ -9,15 +10,21 @@ const SessionsController = {
     console.log("trying to log in");
     const email = req.body.email;
     const password = req.body.password;
-
+    console.log(password);
     User.findOne({ email: email }).then((user) => {
-      if (!user) {
-        res.redirect("/sessions/new");
-      } else if (user.password != password) {
-        res.redirect("/sessions/new");
+      if (user) {
+        bcrypt.compare(password, user.password).then((result) => {
+          if (result) {
+            console.log(`Look here -------- ${user.password}`);
```

Comment:
Same as above -> worth removing just on the off-chance we forget about it

Reply:
These have been removed by Chris.